GoogleDataTransport/GDTCORLibrary/GDTCORRegistrar.m

```diff
@@ -34,6 +34,13 @@
   }
 }
 
+FOUNDATION_EXPORT
```

Comment:
I believe `FOUNDATION_EXPORT` should only be in the header. Also, the above function.

Reply:
Good catch. Done.